From: Michael Fladischer <fladi@debian.org>
Date: Thu, 8 Oct 2015 08:37:05 -0700
Subject: Remove external assets from documentation.

 Upstream includes a PayPal donation form in it's sphinx templates for which the
 submit button used an external image and which was replaced by a simple
 <button/> element. Also a gittip button is loaded inside an <iframe/> which is
 removed. The logo image was loaded from the project website and is replaced
 with the local image file.
Last-Update: 2014-02-11
Forwarded: not-needed

Patch-Name: privacy.patch
---
 docs/_templates/sidebardonations.html | 9 ---------
 docs/conf.py                          | 1 -
 2 files changed, 10 deletions(-)

diff --git a/docs/_templates/sidebardonations.html b/docs/_templates/sidebardonations.html
index 2eebc8e..e69de29 100644
--- a/docs/_templates/sidebardonations.html
+++ b/docs/_templates/sidebardonations.html
@@ -1,9 +0,0 @@
-<iframe src="https://ghbtns.com/github-btn.html?user=celery&repo=celery&type=watch&count=true&size=large"
-  allowtransparency="true" frameborder="0" scrolling="0" width="200px" height="35px"></iframe>
-<div id="donate">
-    <h3>Donations</h3>
-    <p>Please help support this community project with a donation.</p>
-    <a href="https://opencollective.com/celery/donate" target="_blank">
-      <img src="https://opencollective.com/celery/donate/button@2x.png?color=blue"/>
-    </a>
-</div>
diff --git a/docs/conf.py b/docs/conf.py
index f28a5c9..b4419a5 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -14,7 +14,6 @@ globals().update(conf.build_config(
     publisher='Celery Project',
     html_logo='images/celery_512.png',
     html_favicon='images/favicon.ico',
-    html_prepend_sidebars=['sidebardonations.html'],
     extra_extensions=[
         'sphinx_click',
         'sphinx.ext.napoleon',
